libcockatrice_deck_list/libcockatrice/deck_list/deck_list.cpp

```diff
@@ -665,26 +651,18 @@ bool DeckList::deleteNode(AbstractDecklistNode *node, InnerDecklistNode *rootNod
     return false;
 }
 
-static QString computeDeckHash(const InnerDecklistNode *root)
+static QString computeDeckHash(const DeckList &deckList)
 {
+    auto mainDeckNodes = deckList.getCardNodes({DECK_ZONE_MAIN});
```

Comment:
this part made me think of the way I'd like for these to be configurable at some point https://discord.com/channels/733313820499640322/884810697156989008/1448086861800017920

we are not near to that point yet though so this stuff is fine for now, we haven't even talked about how we'd hash other zones yet even